shuup/front/apps/customer_information/views.py

```diff
@@ -5,34 +5,45 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
+import six
 from django.contrib import messages
+from django.contrib.auth.forms import PasswordChangeForm
 from django.contrib.auth.views import password_change
 from django.shortcuts import redirect
 from django.utils.translation import ugettext_lazy as _
-from django.views.generic import FormView
+from django.views.generic import FormView, TemplateView
 
-from shuup.core.models import get_company_contact, get_person_contact
+from shuup.core.models import (
+    get_company_contact, get_person_contact, MutableAddress, SavedAddress
+)
+from shuup.front.views.dashboard import DashboardViewMixin
 from shuup.utils.form_group import FormGroup
 from shuup.utils.importing import cached_load
 
-from .forms import CompanyContactForm, PersonContactForm
+from .forms import CompanyContactForm, PersonContactForm, SavedAddressForm
 from .notify_events import CompanyAccountCreated
 
 
-def change_password(request):
+class PasswordChangeView(DashboardViewMixin, TemplateView):
```

Comment:
Could you tell something about this change in commit message. PRobably could be a separate commit?